packages/deprecated/src/vsce/piranha-subscription.ts

```diff
@@ -0,0 +1,55 @@
+// context.subscriptions.push(
+//   vscode.commands.registerCommand(
+//     "codemod.executeAsPiranhaRule",
+//     async (configurationUri: vscode.Uri) => {
+//       const fileStat = await vscode.workspace.fs.stat(configurationUri);
+//       const configurationUriIsDirectory = Boolean(
+//         fileStat.type & vscode.FileType.Directory,
+//       );
+
+//       if (!configurationUriIsDirectory) {
+//         throw new Error(
```

Comment:
do you want to remove those?

Reply:
yeah, we don't support piranha anymore @arybitskiy

Reply:
at least basically no effort has been put into this during the last 6 months or so

Reply:
i am moving the deprecated code to `deprecated` package that is not in our turbo pipeline, hence is not being built and does not affect deployments in any way. i expect it to stay there for some time until we reach stability so that we don't lose it in case we suddenly want to support those features again